docs/blue-green-migration.md

```diff
@@ -2,24 +2,28 @@
 
 ![Migration Terraform](https://user-images.githubusercontent.com/1868782/117465361-9f83e400-af1f-11eb-8844-b14fefa2c3d2.png)
 
-## Manual Migration Steps
+Every deployment, whether it's to production or a development environment, is a blue-green deployment. We build a copy for the resources that are required to run a second version of the application. When the copy is ready and tested, we switch the client over to run off of the newly deployed color and have the old color available for rolling back.
+
+Additionally, for deployments that require change in the data schema or broad stroke changes to the database in order for the application to function, we build an empty copy of the data stores (DynamoDB and Elasticsearch). Then we perform a migration of information from the old data store into the newly created one. This migration passes all of the data through a lambda to update and verify the data before saving it into a new data store.
+
+## Automated Migration Steps
 
-For Production (`ENV` would be `prod`), this process is entirely manual. For the `test` and `mig` environments, steps 7 and 8 are manual. Additionally, for `stg`, `irs`, or `dev` steps 7 and 8 are manual if the deployment process does not complete the smoke tests step.
+This process is mostly automated. For Production (`ENV` would be `prod`), the final `switch-colors` step is manual. For the rest of the environments that step occurs following a successful smoke tests step.
 
 Here are the steps involved:
 
-1. Setup the destination environments.  Load up the `efcms-ENV-SUFFIX` dynamodb table and figure out the destination version.  If `source-table-version` is `beta`, you'd want to delete the `alpha` dynamodb table and cluster, and vice versa.
+1. Setup the destination environments.  Load up the `efcms-ENV-deploy` dynamodb table and figure out the destination version.  If `source-table-version` is `beta`, you'd want to delete the `alpha` dynamodb table and cluster, and vice versa.
```

Comment:
kind of a nit - should we say "Elasticsearch cluster" rather than just "cluster".

Also may be useful to specify dynamodb for both regions.

Reply:
Also also, is it worth mentioning if you get the "can't delete - replication yadda yadda", you should delete the west cluster first, wait a few mins and then attempt the east cluster?